src/main/java/com/google/codeu/servlets/EventsPageServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.codeu.servlets;
+
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.Message;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles fetching all messages for the events feed. */
+@WebServlet("/events")
+public class MessageFeedServlet extends HttpServlet {
```

Comment:
This needs to be called EventsPageServlet, it's not building when I try to run it bc its saying this is a duplicate class

Reply:
Sorry about that should be fixed!